app/source/js/firebase-helper.js

```diff
@@ -39,6 +39,21 @@ const onNotificationByMonth = callback => {
   };
 };
 
+const onNotificationByYear = callback => {
+  const reference = database.ref('notifications');
+
+  reference
+    .orderByChild('time')
+    .endAt(-time.lastYear()) // Notifications have a negative timestamp in the database
+    .on('value', snapshot => {
```

Comment:
Det her er snakk om ganske mye data. Siden denne er hostet på min personlige Firebase-konto vil jeg gjerne at denne kun kjøres én gang, så `.once()` i stedet for `.on()`. Gidder ikke overraskelses-regning :P